src/ui/common/src/components/integrations/cards/slackCard.tsx

```diff
@@ -0,0 +1,30 @@
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import React from 'react';
+
+import { Integration, SlackConfig } from '../../../utils/integrations';
+
+type Props = {
+  integration: Integration;
+};
+
+export const SlackCard: React.FC<Props> = ({ integration }) => {
+  const config = integration.config as SlackConfig;
+  const channels = JSON.parse(config.channels_serialized) as string[];
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+      <Typography variant="body2">
+        {channels.length > 1 ? (
+          <strong>Channel: </strong>
+        ) : (
+          <strong>Channels:</strong>
+        )}{' '}
+        {channels.join(', ')}
+      </Typography>
+      <Typography variant="body2">
```

Comment:
Can you ensure that there's enough spacing between these please?

I'm not too sure off the top of my head what we should use, but should be something like 4px or 8px here. Dealer's choice.

Reply:
This spacing is consistent with other integrations, do you suggest to change them all?

Reply:
I agree the spacing looks a bit weird on this particular card since we only have 3 lines, but I'm also hesitant to break the convention here.

Reply:
Do you want a task for this?

Reply:
https://linear.app/aqueducthq/issue/ENG-2321/revisit-appearance-in-integration-cards